manifest-editor/components/atoms/Thumbnail.tsx

```diff
@@ -10,17 +11,25 @@ export const ThumbnailImg = styled.img`
 `;
 
 export const Thumbnail: React.FC<{ onClick: () => void }> = ({ onClick }) => {
-  const thumb = useThumbnail({
-    minWidth: 200,
-    minHeight: 200,
-    maxWidth: 300,
-  });
+  let thumb: any;
+  try {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    thumb = useThumbnail({
+      minWidth: 200,
+      minHeight: 200,
+      maxWidth: 300,
+    });
+  } catch (error) {
+    console.warn(error);
+  }
```

Comment:
I think this would be better as an error boundary, with `<div onClick />...` as the fallback. React will complain a lot if that hook passes and then fails (or vice-versa). 

`useThumbnail` should also avoid throwing where possible. What was the error that you saw?